gmso/parameterization/parameterize.py

```diff
@@ -12,63 +12,47 @@ def apply(
+    speedup_by_moltag=False,
+    ignore_params=["improper"],
     remove_untyped=False,
     fast_copy=True,
 ):
     """Set Topology parameter types from GMSO ForceFields.
 
     Parameters
     ----------
-    top: gmso.core.topology.Topology, required
+    top : gmso.core.topology.Topology, required
         The GMSO topology on which to apply forcefields
 
-    forcefields: ForceField or dict, required
+    forcefields : ForceField or dict, required
         The forcefield to apply. If a dictionary is used the keys are labels that match
         the molecule name (specified as a label of site), and the values are gmso ForceField objects that gets applied
         to the specified molecule.
         Note: if a Topology with no molecule is provided, this option will only take
         a ForceField object. If a dictionary of ForceFields is provided, this method will
         fail.
 
-    match_ff_by: str, optional, default="molecule"
+    match_ff_by : str, optional, default="molecule"
         They site's tag used to match the forcefields provided above to the Topology.
         Options include "molecule" and "group". This option is only valid if forcefields are provided
         as a dict.
 
-    identify_connections: bool, optional, default=False
+    identify_connections : bool, optional, default=False
         If true, add connections identified using networkx graph matching to match
         the topology's bonding graph to smaller sub-graphs that correspond to an angle,
         dihedral, improper etc
 
-    identify_connected_components: bool, optional, default=True
+    speedup_by_molgraph: bool, optional, default=True
         A flag to determine whether or not to search the topology for repeated disconnected
         structures, otherwise known as molecules and type each molecule only once.
 
-    use_molecule_info: bool, optional, default=False
-        A flag to determine whether or not to look at site.residue_name to look parameterize
+    speedup_by_moltag : bool, optional, default=False
+        A flag to determine whether or not to look at site.molecule_name to look parameterize
         each molecule only once. Currently unused.
 
-    assert_bond_params : bool, optional, default=True
-        If True, an error is raised if parameters are not found for all system
-        bonds.
-
-    assert_angle_params : bool, optional, default=True
-        If True, an error is raised if parameters are not found for all system
-        angles.
-
-    assert_dihedral_params : bool, optional, default=True
-        If True, an error is raised if parameters are not found for all system
-        proper dihedrals.
-
-    assert_improper_params : bool, optional, default=False
-        If True, an error is raised if parameters are not found for all system
-        improper dihedrals.
+    ignore_params : list, optional, default=["impropers"]
+        Skipping the checks that make sure all connections (in the list) have a connection types.
```

Comment:
I think this doc string should list out all possible options that can go in this list just to help avoid any confusion.